save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/utils/Warning.kt

```diff
@@ -108,16 +109,26 @@ internal fun String.extractWarning(warningRegex: Regex,
     "SwallowedException")
 internal fun String.getLineNumber(warningRegex: Regex,
                                   lineGroupIdx: Int?,
-): Int? {
+                                  placeholder: String,
+): Pair<Int?, Boolean>? {
```

Comment:
Can `first` be null if pair is not null?

Reply:
Yes, if lineGroupIdx = null